google-ads-examples/src/main/java/com/google/ads/googleads/examples/campaignmanagement/UpdateCampaignCriterionBidModifier.java

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.campaignmanagement;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.lib.utils.FieldMasks;
+import com.google.ads.googleads.v2.common.DeviceInfo;
+import com.google.ads.googleads.v2.enums.DeviceEnum.Device;
+import com.google.ads.googleads.v2.services.CampaignCriterionOperation;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.CampaignCriterion;
+import com.google.ads.googleads.v2.services.CampaignCriterionServiceClient;
+import com.google.ads.googleads.v2.services.MutateCampaignCriteriaResponse;
+import com.google.ads.googleads.v2.services.MutateCampaignCriterionResult;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FloatValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/** Sets a bid modifier for the mobile platform on given campaign. */
+public class UpdateCampaignCriterionBidModifier {
+
+  private static final float BID_MODIFIER = 1.5f;
+
+  private static class UpdateCampaignCriterionBidModifierParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private Long customerId;
+
+    @Parameter(names = ArgumentNames.CAMPAIGN_ID, required = true)
+    private Long campaignId;
+
+    @Parameter(names = ArgumentNames.CRITERION_ID, required = true)
```

Comment:
Sorry, one last request: could you make bid modifier a parameter as well? You could have it default to 1.5, as we do in AddAdGroupBidModifier.